src/main/kotlin/net/atos/zac/app/signaleringen/converter/RESTSignaleringInstellingenConverter.kt

```diff
@@ -13,9 +13,11 @@ import net.atos.zac.signalering.model.SignaleringInstellingen
 import net.atos.zac.signalering.model.SignaleringTarget
 import java.util.stream.Collectors
 
-class RESTSignaleringInstellingenConverter @Inject constructor(
-    private var signaleringenService: SignaleringenService
-) {
+class RESTSignaleringInstellingenConverter {
```

Comment:
Hmm that was not really me, but IntelliJ converter thingy. I think it is because it thinks `SignaleringenService` is not usable (proxyable)

Will fix that